packages/shopify-app-express/jest.config.ts

```diff
@@ -0,0 +1,10 @@
+import type {Config} from 'jest';
+
+import baseConfig from '../../config/tests/jest.config';
+
+const config: Config = {
+  ...baseConfig,
+  testTimeout: 35000,
```

Comment:
Any reason for increasing the testTimeout for these ones?

Reply:
I remember I had to do that, let me see if we can not.